package com.kotlin.blog.common.exception

class LoginFailureException() : RuntimeException()
